### PR TITLE
Header styling (Story1task1)

### PR DESCRIPTION
**Pull request by Jollion81**
Header styling was finished (Story1task1)

### CODE REVIEW

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react"
+
+export default function Header() {
+    const [unpaid, setUnpaid] = useState(0)
+    const [data, setData] = useState([])
```
Comment (FloCRC) [inline src/components/Header/index.jsx:5]: This constant isn't being used for anything, and can be removed.

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    useEffect(() => {
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices')
+            .then(res => res.json())
+            .then((invoices) => {
+                setData(invoices.data)
```
Comment (FloCRC) [inline src/components/Header/index.jsx:11]: This will not be required once the const is removed.

**Review by FloCRC — CHANGES_REQUESTED** [review]
The const on line 5, and setting of state on line 11, are not refereed to again, I think. So, aren't needed,

**Comment by Xtrellah** [comment]
need to merge to story1 not main btw

**Review by FloCRC — APPROVED** [review]

**Pull request merged**